src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -329,20 +340,21 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     val fileContent =
       s"""
          |#!/bin/bash
+         |export _JAVA_OPTIONS=-Djava.io.tmpdir=$JesCromwellRoot/tmp
```

Comment:
Right I keep adding this one back when I need to start GotC jobs but sometimes forget to remove it  It would be nice if we could get it merged in btw as the single sample won't run without it for large files.